dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/EventQueryParams.java

```diff
@@ -1324,7 +1324,7 @@ public Builder withOutputType( EventOutputType outputType )
             return this;
         }
 
-        public Builder withEventStatus( EventStatus eventStatus )
+        public Builder withEventStatus( Set<EventStatus> eventStatus )
```

Comment:
minor: should we rename method to "withEventStatuses".
Same comment applies for programStatus->programStatuses

Reply:
.. or maybe programStatusList or Set

Reply:
Done